playbook/app/pb_kits/playbook/pb_caption/_caption.scss

```diff
@@ -1,19 +1,38 @@
+@import "../tokens/colors";
 @import "caption_mixin";
 
-[class^=pb_caption_kit] {
-  @include caption;
+[class^="pb_caption_kit"] {
+	@include caption;
+
+	&[class^="pb_caption_kit_lg"] {
+		@include caption_lg;
+	}
+
+	&[class^="pb_caption_kit_xs"] {
+		@include caption_xs;
+		&[class*="_link"] {
+			color: $primary;
+		}
+	}
+
+	&[class*="light"] {
+		color: $text_lt_light;
+	}
 
-  &[class^=pb_caption_kit_lg] {
-    @include caption_lg;
-  }
 
-  &[class^=pb_caption_kit_xs] {
-    @include caption_xs;
-    &[class*=_link]  {
-      color: $primary;
+	&[class*="default"] {
+		color: $text_lt_default;
+	}
+
+	&[class*="link"] {
+		color: $primary;
+	}
+
+	&[class*=dark] {
```

Comment:
This is missing quotes.. My suggestion here is to follow the standards that other scss files have established regarding these quotes. Mix and match is undesirable code style.